pkg/skaffold/schema/latest/config.go

```diff
@@ -225,7 +225,7 @@ type LocalBuild struct {
 	UseDockerCLI bool `yaml:"useDockerCLI,omitempty"`
 
 	// UseBuildkit use BuildKit to build Docker images.
-	UseBuildkit bool `yaml:"useBuildkit,omitempty"`
+	UseBuildkit bool `yaml:"useBuildkit,omitempty" default:"true"`
```

Comment:
We've generally been using `*bool` instead, like `Push` above.

Reply:
ok